### ISSUE TITLE
Suggestion for Celluloid

### ISSUE THREAD

**Issue by dilumn** [dilumn/celluloid#55]
I got this idea at the Ruby Conference I participated in India. Can't we print some Celluloid Laptop Stickers to distribute among developers? Hope it'll not cost much.

**Comment by digitalextremist** [comment]
Yeah. I'm working on stickers, t-shirts, and possibly even embroidered hats and polos. Also decals for glass, like cars and office doors.

**Comment by dilumn** [comment]
Great..... Waiting to see all of them in future.....

**Comment by digitalextremist** [comment]
Do you mind researching die-cut stickers... say 1.5in tall by 6in wide?

**Comment by dilumn** [comment]
I request a quotation from a printing company here. Do you want me to search for a International printing place?

**Comment by digitalextremist** [comment]
Yes, one that's online and in the US

**Comment by dilumn** [comment]
I went through several websites & seems like this is the best among them.

https://www.stickermule.com/products/die-cut-stickers

Their Price is reasonable & they can ship the Stickers within 5 business days (Free Shipping in US)

**Comment by digitalextremist** [comment]
That's perfect. Thanks

**Comment by digitalextremist** [comment]
Nice, I can get a huge wall graphic too.

**Comment by digitalextremist** [comment]
Also, we can add the sticker to the Open Source sticker pack once it's been ordered once, I think.

**Comment by dilumn** [comment]
What's Open Source sticker pack? I didn't get it

**Comment by digitalextremist** [comment]
For example, this collection:

https://www.stickermule.com/marketplace/collections/open-source-stickers

I just bought 5 of the jRuby and 5 of the Mint stickers, for example... so once I order 50 Celluloid stickers, they can go in that collection too.

**Comment by digitalextremist** [comment]
I bought all 4 GitHub sticker packs recently, but I forgot Sticker Mule did them for GitHub. The quality is awesome.

**Comment by dilumn** [comment]
Just now saw this collection. It's awesome. I'm going to buy a GitHub sticker pack now too.

**Comment by dilumn** [comment]
Ordered a Celluloid logo sticker :)

**Comment by digitalextremist** [comment]
Yep, I saw :)

**Comment by digitalextremist** [comment]
There's another one now.

I'm putting both stickers on celluloid.io ...

If you could look up a good source for apparel that'd be helpful:

* T-Shirts
* Polos
* Hats
* etc.
